index.js

```diff
@@ -71,5 +73,67 @@ export default function icomesh(order = 4) {
         vertices[i + 2] *= m;
     }
 
-    return {vertices, triangles};
+    if (!uvMap) return {vertices, triangles};
+
+    // uv mapping
+    const uv = new Float32Array((numVertices + numDuplicates) * 2);
+    for (let i = 0; i < numVertices; i++) {
+        uv[2 * i + 0] = Math.atan2(vertices[3 * i + 2], vertices[3 * i]) / (2 * Math.PI) + 0.5;
```

Comment:
Better use `x * (1.0 / (2 * Math.PI))` instead `x / (2 * Math.PI)`. Usually even LLVM can't optimize this case without `-fast-math` flag because it not strictly equivalent but could slightly sppedup.